stl/inc/complex

```diff
@@ -1585,21 +1585,23 @@ _NODISCARD complex<_Ty> log(const complex<_Ty>& _Left) {
 
 // FUNCTION TEMPLATE pow
 template <class _Ty>
-_NODISCARD complex<_Ty> pow(const complex<_Ty>& _Left, int _Right) {
-    complex<_Ty> _Tmp = _Left;
-    auto _Count       = static_cast<unsigned int>(_Right);
+_NODISCARD complex<_Common_float_type_t<_Ty, double>> pow(const complex<_Ty>& _Left, int _Right) {

```

Comment:
Pre-existing: `_Right` isn't modified, so it should be `const`. (We try to pass parameters by `const` value whenever possible in new or updated code; old code is inconsistent about this.)